Mathlib/Algebra/Order/Nonneg/Lattice.lean

```diff
@@ -46,8 +46,11 @@ instance instDenselyOrdered [Preorder α] [DenselyOrdered α] {a : α} :
 
 /-- If `sSup ∅ ≤ a` then `{x : α // a ≤ x}` is a `ConditionallyCompleteLinearOrder`. -/
 protected noncomputable abbrev conditionallyCompleteLinearOrder [ConditionallyCompleteLinearOrder α]
-    {a : α} : ConditionallyCompleteLinearOrder { x : α // a ≤ x } :=
-  { @ordConnectedSubsetConditionallyCompleteLinearOrder α (Set.Ici a) _ ⟨⟨a, le_rfl⟩⟩ _ with }
+    {a : α} : ConditionallyCompleteLinearOrder { x : α // a ≤ x } where
+  __ := @ordConnectedSubsetConditionallyCompleteLinearOrder α (Set.Ici a) _ ⟨⟨a, le_rfl⟩⟩ _
+  decidableEq := inferInstance
+  decidableLE := inferInstance
+  decidableLT := inferInstance
```

Comment:
Are there other data fields of `LinearOrder` we should care about here? E.g., `Ord`? BTW, should we fix `ordConnectedSubsetConditionallyCompleteLinearOrder` to use the right `decidableEq` etc instead?

Reply:
Yes, I've created #23514 to try and resolve this bigger issue.

Reply:
Misdiagnosed, the actual issue was #23750